cmd/genbindings/emitcabi.go

```diff
@@ -600,11 +604,54 @@ func getCppZeroValue(p CppParameter) string {
 		return "0"
 	} else if p.ParameterType == "bool" {
 		return "false"
+	} else if p.ParameterType == "void" {
+		return ""
 	} else {
 		return p.RenderTypeQtCpp() + "()"
 	}
 }
 
+func getCabiZeroValue(p CppParameter) string {
```

Comment:
can probably get away with plain `return {};` for all the below types except `void`, simplifying the generator a little.